dropwizard-db/src/test/java/io/dropwizard/db/DataSourceFactoryTest.java

```diff
@@ -47,6 +48,18 @@ private ManagedDataSource dataSource() throws Exception {
     }
 
     @Test
+    public void testInitialSizeIsZero() throws Exception {
+        factory.setUrl("nonsense invalid url");
+        factory.setInitialSize(0);
+        ManagedDataSource dataSource = factory.build(metricRegistry, "test");
+        try {
+            dataSource.start();
```

Comment:
Surrounding the statement with try-catch doesn't gain anything. The test will fail anyway, if any uncaught exception occurs.